packages/network-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [21.1.0]
+
+### Changed
+
+- Bump `@metamask/eth-json-rpc-provider` from `^4.1.4` to `^4.1.5` ([#4798](https://github.com/MetaMask/core/pull/4798))
```

Comment:
@bergeron Noting that this change was merged `BREAKING` (https://github.com/MetaMask/core/pull/4769) but released as non-breaking. Is there some reason this was still released as `21.1.0` ad not `22.0.0`?

Reply:
Good catch. Even if it was non-breaking, it certainly should have been listed here as a change.